pkg/templates/charts/toggle/console/templates/console-search-configmap.yaml

```diff
@@ -30,5 +30,11 @@ data:
         "name": "Created last hour",
         "description": "Search for resources created within the last hour",
         "searchText": "created:hour"
+      },
+      {
+        "id": "search.suggested.virtualmachines.name",
+        "name": "Virtual Machines",
+        "description": "Show virtual machines under management",
```

Comment:
Can you change the description to: 
```
Show virtual machine resources.
```
I want to avoid _under management_ because ACM is not directly managing the VMs.